packages/editor/src/components/block-list/style.scss

```diff
@@ -201,10 +201,14 @@
 
 	// Warnings
 	&.has-warning .editor-block-list__block-edit {
-		> :not(.editor-warning) {
+		> * {
```

Comment:
That's a scary selector! 😅

Can we explain/contextualise this one, just so folks know why it is?

Reply:
Yes indeed. It's *any direct descendant of .editor-block-list__block-edit, for a block that has a warning*, and then down below we make an exception for the .editor-warning. 

The contents of a block that has a warning is basically locked from being interacted with. 

I'm not sure how to do it better — a different fix for this would be to do `:not(.editor-warning)` instead of `> :not(.editor-warning)` (as the warning is no longer a direct descendant). But a recent ticket suggests to not use `:not`, so the wildcard + override is sort of the inversion of that. 

I suspect you already knew all this, and wasn't asking for clarification but just a CSS comment. I suppose writing it out helped me formulate it. How about:

`// When a block has a warning, you shouldn't be able to manipulate the contents.`

`// Allow the warning action buttons to be manipulable.`

Something in that vein?

Reply:
Works for me! 👍

Can you still interact with it using a keyboard even though these CSS props are set? I should think no, but wanted to check.

Reply:
@jasmussen
>But a recent ticket suggests to not use `:not`

I'm curious: which ticket, and why?

Reply:
I personally prefer `:not` over wildcard. I'd be very cautious about "absolute rules". `:not` can be good if used properly.

Reply:
This is the issue in question: #11677

Reply:
> Can you still interact with it using a keyboard even though these CSS props are set? I should think no, but wanted to check.

Yes. With the block focused, it's 5 tab presses to get to the resolve button.

Reply:
Sorry, I meant can you access the underlying content that is disabled with `pointer-events`? But I guess it was disabled this way already and is cool. Just ignore me 😉

Reply:
> Sorry, I meant can you access the underlying content that is disabled with pointer-events? But I guess it was disabled this way already and is cool. Just ignore me 😉

Well if that underlying content has buttons or other focusable elements, tab should be able to reach it if we don't do anything with JS to skip that, I can't recall the full details. But to answer the meta question: yes, pointer-events "only" disables mouse interaction.